app/main/views/users.py

```diff
@@ -277,6 +278,7 @@ def export_users_for_framework(framework_slug):
             else:
                 application_result = 'pass' if sf.on_framework else 'fail'
             framework_agreement = bool(sf.agreement_returned_at)
+            variations_agreed = ', '.join(sf.agreed_variations.keys()) if sf.agreed_variations else ''
```

Comment:
Just a thought, instead of returning an empty string if there are no agreed variations, it might be easier to read for an admin if the string 'None' is returned here, or something similar.